BlogTemplate/Pages/Edit.cshtml

```diff
@@ -26,14 +26,20 @@
             <input type="hidden" name="updateslug" id="updateslug" value="false" />
             <div class="col-md-12 form-group">
                 <label for="Body" class="primaryField">Body</label>
-                <textarea rows="10" class="form-control" placeholder="" asp-for="EditedPost.Body"></textarea>
+                <textarea rows="10" class="form-control" placeholder="" asp-for="EditedPost.Body" id="post-body"></textarea>

```

Comment:
I don't think the `id=` is needed since you're using the `asp-for` tag helper

Reply:
Oh, is this for the js that handles when you upload new files?  Do the tag helper and the id conflict?

Reply:
@jimmylewis It doesn't seem to be conflicting, but is there a way to test some situation where this could break?